README.md

```diff
@@ -129,10 +129,11 @@
 * [pre-commit-hooks.nix](https://github.com/cachix/pre-commit-hooks.nix) - Run linters/formatters at commit time and on your CI.
 * [rnix-lsp](https://github.com/nix-community/rnix-lsp) - A syntax-checking language server for Nix.
 * [robotnix](https://github.com/danielfullmer/robotnix) - A declarative and reproducible build system for Android (AOSP) images.
+* [runix](https://github.com/flox/runix) - A typesafe Rust interface to the nix CLI.
```

Comment:
Capitalize Nix please.

I kinda wonder if this should go in the "Rust" subsection of "Programming Languages", since it's a library crate...